### PR TITLE
fix: resolve conflicts with UI/UX improvements

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/components/earn-add/earn-add-confirm/EarnAddConfirm.tsx [hunk 1]
```diff
@@ -67,7 +68,8 @@ const EarnAddConfirm: React.FC<EarnAddConfirmProps> = ({
 
       <Button
         text="Confirm Add Position"
-        onClick={confirm}
+        onClick={feeInfo.errorMsg ? undefined : confirm}
+        disabled={!!feeInfo.errorMsg}
```
Comment (jinoosss) [inline packages/web/src/components/earn-add/earn-add-confirm/EarnAddConfirm.tsx:72]: Define the onClick function.
It doesn't look good to intentionally use an undefined value.

File: packages/web/src/components/home/token-info/TokenInfo.styles.ts
Comment (jinoosss) [inline packages/web/src/components/home/token-info/TokenInfo.styles.ts:1]: Please remove the comma on line 138.

File: packages/web/src/components/pool/my-liquidity/MyLiquidity.tsx [hunk 2]
```diff
@@ -115,13 +124,12 @@ const MyLiquidity: React.FC<MyLiquidityProps> = ({
                     connected={connected}
                     tokenPrices={tokenPrices}
                   />
-                ))}
-              </div>
+                ))}              </div>
```
Comment (jinoosss) [inline packages/web/src/components/pool/my-liquidity/MyLiquidity.tsx:127]: Please reformat your code.

File: packages/web/src/components/pool/my-liquidity-content/MyLiquidityContent.tsx [hunk 3]
```diff
@@ -379,34 +387,51 @@ const MyLiquidityContent: React.FC<MyLiquidityContentProps> = ({
...
+    const swapFee = aprRewardInfo?.SWAP_FEE;
     const sumUSD =
-      temp?.reduce((accum, current) => accum + current.accuReward1DPrice, 0) ||
+      swapFee?.reduce((accum, current) => accum + current.accuReward1DPrice, 0) ||
       0;
     if (sumUSD > 0 && sumUSD <= 0.01) return "<$0.01";
 
-    return formatUsdNumber(`${sumUSD}`, 2, true);
-  }, [aprRewardInfo?.SWAP_FEE]);
+    return toPriceFormat(`${"0.202"}`, {
```
Comment (jinoosss) [inline packages/web/src/components/pool/my-liquidity-content/MyLiquidityContent.tsx:398]: What does 0.202 mean?

File: packages/web/src/constants/skeleton.constant.ts [hunk 4]
```diff
@@ -391,107 +391,157 @@ export const POOL_INFO_MOBILE = {
   ],
 };
 
-export const ASSET_INFO = {
+// export const ASSET_TD_WIDTH = [300, 300, 250, 250, 120, 138];
+// export const TABLET_ASSET_TD_WIDTH = [220, 240, 190, 190, 120, 138];
+// export const MOBILE_ASSET_TD_WIDTH = [230, 230, 195, 195, 90, 120];
```
Comment (jinoosss) [inline packages/web/src/constants/skeleton.constant.ts:396]: Is this a required comment?

File: packages/web/src/containers/highest-aprs-card-list-container/HighestAprsCardListContainer.tsx [hunk 5]
```diff
@@ -9,6 +9,7 @@ const HighestAprsCardListContainer: React.FC = () => {
   const router = useRouter();
   const { breakpoint } = useWindowSize();
   const { higestAPRs } = usePoolData();
+  console.log("🚀 ~ higestAPRs:", higestAPRs);
```
Comment (jinoosss) [inline packages/web/src/containers/highest-aprs-card-list-container/HighestAprsCardListContainer.tsx:12]: Please remove console log.

File: packages/web/src/hooks/decrease/use-decrease-position-modal.tsx [hunk 6]
```diff
@@ -135,7 +148,9 @@ export const useDecreasePositionModal = ({
             }),
           );
         }, 1000);
-        router.back();
+
+        console.log("2384729834789237");
```
Comment (jinoosss) [inline packages/web/src/hooks/decrease/use-decrease-position-modal.tsx:152]: Please remove console log.

File: packages/web/src/hooks/wallet/use-wallet.ts [hunk 7]
```diff
@@ -91,64 +90,76 @@ export const useWallet = () => {
...
+      setLoadingConnect("loading");
+      const res = await accountRepository.switchNetwork(CHAIN_ID);
+      if (res.code === 0) {
+        const account = await accountRepository.getAccount();
+        setWalletAccount(account);
+        accountRepository.setConnectedWallet(true);
+      }
+      setLoadingConnect("done");
+    } catch (error) {
+      console.log("🚀 ~ switchNetwork ~ error:", error);
```
Comment (jinoosss) [inline packages/web/src/hooks/wallet/use-wallet.ts:111]: Please remove console log.

File: packages/web/src/hooks/wallet/use-wallet.ts [hunk 8]
```diff
@@ -91,64 +90,76 @@ export const useWallet = () => {
...
+        setLoadingConnect("error");
       }
-      setWalletAccount(account);
-      accountRepository.setConnectedWallet(true);
-      setLoadingConnect("done");
-    } else {
-      accountRepository.setConnectedWallet(false);
-      setLoadingConnect("error");
+    } catch (error) {
+      console.log("🚀 ~ connectAccount ~ error:", error);
```
Comment (jinoosss) [inline packages/web/src/hooks/wallet/use-wallet.ts:160]: Please remove console log.

File: packages/web/src/pages/earn/pool/[pool-path]/add.tsx [hunk 9]
```diff
@@ -38,19 +45,55 @@ export default function EarnAdd() {
...
-      exchangeRateGraph={<ExchangeRateGraphContainer />}
-      footer={<Footer />}
-    />
+    <>
+      <SEOHeader
+        title={title}
+        pageDescription={seoInfo.desc()}
+        ogTitle={seoInfo?.ogTitle?.()}
+        ogDescription={seoInfo?.ogDesc?.()}
+      />
```
Comment (jinoosss) [inline packages/web/src/pages/earn/pool/[pool-path]/add.tsx:83]: This is the code that goes into each page.
Isn't there a common way to handle it?

**Review by jinoosss — COMMENTED** [review]
I left some minor comments.

File: packages/web/src/components/pool/my-liquidity-content/MyLiquidityContent.tsx (hunk 3 above)
Comment (phucvarmeta) [reply packages/web/src/components/pool/my-liquidity-content/MyLiquidityContent.tsx:398]: That's testing code

File: packages/web/src/constants/skeleton.constant.ts (hunk 4 above)
Comment (phucvarmeta) [reply packages/web/src/constants/skeleton.constant.ts:396]: removed

Comment (phucvarmeta) [reply packages/web/src/constants/skeleton.constant.ts:396]: removed

File: packages/web/src/pages/earn/pool/[pool-path]/add.tsx (hunk 9 above)
Comment (phucvarmeta) [reply packages/web/src/pages/earn/pool/[pool-path]/add.tsx:83]: I think they have same component with different data, but i'll check it after PR

Comment (jinoosss) [reply packages/web/src/pages/earn/pool/[pool-path]/add.tsx:83]: Okay, I'll leave this comment open.

**Review by jinoosss — APPROVED** [review]

**Pull request merged**